src/rotest/cli/client.py

```diff
@@ -178,6 +184,17 @@ def main(*tests):
     if len(tests) == 0:
         tests = discover_tests_under_paths(options.paths)
 
+    if options.filter not in (None, "*"):
```

Comment:
i think the "*" is an overkill. you wouldn't cover "* or *" or other pass-all filters